### ISSUE TITLE
Neovim support

### ISSUE THREAD

**Issue by fresh2dev** [brandon1024/fern-renderer-nf.vim#1]
Thank for you for the fantastic addition to my vim config! Quite the enhancement to stock Fern, and essential for navigating large trees.

I often hop from vim/neovim interchangeably. I read the blurb about this plugin not being compatible with neovim (and verified it does not work). The [linked PR](https://github.com/lambdalisue/fern.vim/pull/465) appears to have been merged into Fern.vim. Any chance this could unblock neovim support for this project?

Again, stellar enhancement; way cool. :pray:

**Comment by fresh2dev** [comment]
I just slapped this in my vimrc and now Vim looks better than Neovim :star_struck: 

```
if has('nvim')
    let g:fern#renderer#nerdfont#indent_markers = 1
    let g:fern#renderer = "nerdfont"
else
    let g:fern#renderer = 'brandon1024/fern-renderer-nf.vim'
endif
```

**Comment by brandon1024** [comment]
Thanks @fresh2dev for your support :-) I'm glad to see this project used by others aside from me!

Supporting Neovim is super tricky because textprops [aren't supported by neovim](https://github.com/neovim/neovim/issues/13661). It's a bit of a bummer that neovim is diverging so much from vim, it's quite hard to support both editors these days.

That PR that I linked to in the README added support to Fern for rendering with textprops. If we wanted to support neovim here, we'd also need to update Fern to support extmarks. I don't use neovim so the effort there was too high for me.

With that said, the maintainer of Fern is super friendly and helpful and would probably accept a PR for such a feature. If you or another developer added such a feature in Fern, I'd be happy to update this plugin :-)
